internal/common/types.go

```diff
@@ -54,6 +54,17 @@ func (FetchArgs) Kind() string {
 	return "fetch"
 }
 
+type CollectArgs struct {
+	Scheme    string `json:"scheme"`
+	Host      string `json:"host"`
+	Path      string `json:"path"`
+	JSON      string `json:"json"`
```

Comment:
Yeah. This field. Lets either call this `JSON` or `RawData`... it shows up in different ways. (Per comment above.) I'm inclined toward `JSON`, because that's what it is. Your call.